Doc/library/xml.etree.elementtree.rst

```diff
@@ -166,6 +166,11 @@ data but would still like to have incremental parsing capabilities, take a look
 at :func:`iterparse`.  It can be useful when you're reading a large XML document
 and don't want to hold it wholly in memory.
 
+Where _immediate_ feedback through events is wanted, calling method
```

Comment:
One asterisk is used for emphasis. https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup
```suggestion
Where *immediate* feedback through events is wanted, calling method
```

Reply:
@serhiy-storchaka my bad, my brain works markdown. Fixed.